src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -600,6 +600,9 @@ protected function lock(Request $request, Response $entry)
      */
     protected function store(Request $request, Response $response)
     {
+        if (!$response->headers->has('Date')) {
```

Comment:
I wanted to explicitly call `setDate` to clarify what the code is doing even though one line containing the following would have the same effect:

```php
$response->getDate();
```